### ISSUE TITLE
IV HOG only LCE Addition

### ISSUE THREAD

**Issue by Prometheus0000** [GTNewHorizons/GT-New-Horizons-Modpack#6890]
I made one, even though the code is not especially great. It's mostly just changing a few numbers. This is to make HOG more useful, and fill the gap between IV and LuV power production. See https://github.com/GTNewHorizons/GT5-Unofficial/pull/335.

- HOG only, no other fuels, uses 1/4 L/t depending on boosting
- Uses LOX instead of O2, and uses 8x, so 16L/t instead of 2L/t, so 320L/s instead of 40L/s
- Uses 8x Lubricant
- Puts out 8x pollution (480 vs 3840)
- Does not get a fuel bonus like the LCE, which used only 2/3 the fuel when boosting
- Efficiency goes to 400% instead of 300%
- Outputs 8192/32768 EU

- Materials to build are basically the same, but +1 tier
- The TS gearbox in code is the assline casing, so I just kept the titanium one

Suggestions on whether this is balanced of if it should be altered are welcome. Assuming I can do it.

**Comment by 0lafe** [comment]
Imo this is really boring, and not what HOG needs. Simply adding another gen the next tier is very LV-HV power gen mentality, and not very fun. HOG also isn't really IV or LuV at all, making it a bit of a weird fuel to push that far. Imo just don't do this lol. HOG doesn't really need any boosts

Definitely higher eff than that given 1-4L/t. LCE is only 1.5 right? This seems to be almost a 5x boost, which is crazy. Definitely don't increase eff if you're doing this at all. If you really need to make a HOG gen, I'd strongly recommend lowering efficiency when raising throughput. Seems to be a crazy boost to HOG. Basically makes the gen all you need to upgrade this to very easy pre fusion power. I'd stick to 100-150%, maybe giving it an LRE type throughput/efficiency curve if you want it to function a little more interestingly. 

TL:DR HOG is cool, but not up to fusion cool. Don't boost the fuel gen that heavily/at all, it doesn't need it. LCE but with IV parts is really boring, and seems to be basically just LV-HV gen tiering, but more rewards. Bad payout for the effort given, needs to be more punishing/not exist

**Comment by Prometheus0000** [comment]
Also see #7134. You should have posted that here, actually.

**Comment by GeicowithaBango** [comment]
I suggest to use Liquid air as coolant instead of liquid Oxygen due to two ideas of mine: 
1) Liquid Oxygen is more expensive to cool down. both gases are easy accessible its just the energy needed to cool it down
2) the second more important part is that liquid air is already needed to get the noble gases, so its needed in big quantities, yet not continuously. that definitely feels unrewarding, which  would be changed.
